### DISCUSSION TITLE
Changing volume in-map not working as expected

### DISCUSSION THREAD

**Discussion by Pinto78** [ppy/osu#26010] [Ideas] [answered]
In stable, when you enter the map, if you click the up arrow or down arrow you can change the volume of the map. However in lazer it forces you to use ALT + UP which also forces the player to take both hands out of they're peripherals. This is uncorfortable. Also there is no hotkey configuration for in-game changing of volume so the feature that I described of stable is simply absent in lazer (that i've noticed at least..).

**Comment by peppy** [comment, marked as answer]
works fine here if you set it to this

![2023-12-22 02 32 29@2x](https://github.com/ppy/osu/assets/191335/138387eb-6cec-4fe0-b41d-a3d3fcfa35cb)

**Reply by Pinto78** [reply]
Sorry then, just tried this and yes its fine. I didnt think it would work on song select after changing this here. 

Also something else that I noticed meanwhile, up down arrows in stable in game changes the volume by count of 5, any way to replicate this is lazer?

**Reply by peppy** [reply]
it will accelerate if you press it faster
